resources/views/page_builder/brand_overview.blade.php

```diff
@@ -0,0 +1,53 @@
+@php
+    $brands = \Statamic\Facades\Entry::query()
+        ->where('collection', 'brands')
+        ->where('site', \Statamic\Facades\Site::current()->handle())
+        ->orderBy('title')
+        ->get();
+
+    $grouped = $brands->groupBy(function(\Statamic\Eloquent\Entries\Entry $item, int $key) {
+        return $item->title[0];
+    });
+@endphp
+
+<div class="lg:container">
+    @if($include_brand_list)
+        <div>
+            <span>@lang('Brands')</span>
+            <ul class="flex flex-col">
+                @foreach($brands as $brand)
+                    <li>
+                        <a href="{{ $brand->url() }}">{{ $brand->title }}</a>
+                    </li>
+                @endforeach
+            </ul>
+        </div>
+    @endif
+    <div class="flex flex-col">
+        @if($page->title)
+            <h1>{{ $page->title }}</h1>
+        @endif
```

Comment:
Don't think we should include this. If a brands overview page is build like with content before this component; most likely the page title is included there.